spec/lib/treasury/storage/postgre_sql/db_spec.rb

```diff
@@ -0,0 +1,126 @@
+describe Treasury::Storage::PostgreSQL::Db do
+  let(:storage_params) do
+    {
+      table: 'test_table',
+      key: 'object_id',
+      key_type: :integer,
+      fields: {field1: {type: :integer}, field2: {type: :varchar}}
+    }
+  end
+  let(:storage) { described_class.new(storage_params) }
+  let(:connection) { ActiveRecord::Base.connection }
+
+  before do
+    allow(storage).to receive(:connection).and_return(connection)
+  end
+
+  describe '#bulk_write' do
+    let(:chomp_sql) { ->(text) { text.gsub(/[\s\n]+/, '') } }
+    let(:sql_for_update) do
+      -> (fields_list, values) do
```

Comment:
Layout/SpaceInLambdaLiteral: Do not use spaces between -> and ( in lambda literals.